README.md

```diff
@@ -42,3 +42,81 @@ Be sure to review all `*.test.*` files for implementation details.
 ```
 
 Reward formula: `((count * wordValue) * (score * formattingMultiplier) * n) * relevance + task.reward = total`
+
+## Plugin configuration
+
+Here is a possible valid configuration to enable this plugin.
+
+
+```yaml
+plugin: ubiquibot/conversation-rewards
+with:
+    evmNetworkId: 100
+    evmPrivateEncrypted: "encrypted-key"
+    incentives:
+      enabled: true
```

Comment:
add `generateOnMissingPriceLabel` to the example

Reply:
I think default should be `true` in case the property doesn't exist.